core/src/main/scala/org/apache/spark/deploy/master/MasterMessages.scala

```diff
@@ -36,7 +36,7 @@ private[master] object MasterMessages {
 
   case object CompleteRecovery
 
-  case object RequestWebUIPort
```

Comment:
this is actually for tests. Note that we already break older akka clients regardless because we need a way to propagate the REST port to the web UI